docs/release_notes.rst

```diff
@@ -10,7 +10,7 @@ This is major release and contains breaking changes. Please read the notes below
 
 **Polymorphism**
 
-This release introduces polymorphism support via ``DiscriminatorAttribute``.
+This release introduces :ref:`polymorphism` support via :py:class:`DiscriminatorAttribute <pynamodb.attributes.DiscriminatorAttribute`.
```

Comment:
is this right (syntax)?

Reply:
Yep, should be. Github diff doesn't work understand sphinx